libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/Options.kt

```diff
@@ -226,18 +229,85 @@ interface CommonCodegenOpt {
   val rootPackageName: String?
 
   /**
-   * Whether to decapitalize fields. This is useful if your schema has fields starting with an uppercase as
+   * Whether to decapitalize fields  (for instance `FooBar` -> `fooBar`). This is useful if your schema has fields starting with an uppercase as
    * it may create name clashes with models that use PascalCase
+   *
+   * Default: false
    */
   val decapitalizeFields: Boolean?
   /**
-   * Whether to suffix operation name with 'Query', 'Mutation' or 'Subscription'
+   * When true, the operation class names are suffixed with their operation type like ('Query', 'Mutation' ot 'Subscription').
+   * For an example, `query getDroid { ... }` GraphQL query generates the 'GetDroidQuery' class.
+   *
+   * Default value: true
    */
   val useSemanticNaming: Boolean?
   /**
-   * Which methods to auto generate on models, fragments, operations, and input objects
+   * Specifies which methods will be auto generated on operations, models, fragments and input objects.
+   *
+   * Pass a list of the following:
+   *
+   * - "equalsHashCode" generates `equals` and `hashCode` methods that will compare generated class properties
+   * - "toString" generates a method that will print a pretty string representing the data in the class
+   * - "copy" (Kotlin only) generates a method that will copy the class with named parameters for
```

Comment:
Never noticed before, but this sentence is missing a few words?

Reply:
Added in https://github.com/apollographql/apollo-kotlin/pull/5591/commits/0246deff91d3e4785e6aad3a924a1ab6915936e2